External/Plugins/HaXeContext/Context.cs

```diff
@@ -1668,8 +1668,26 @@ public override bool BuildCMD(bool failSilently)
             RunCMD(command);
             return true;
         }
+
         #endregion
 
+        #region haxelib
+
+        internal void Install(List<string> libraries)
+        {
+            var haxePath = PathHelper.ResolvePath(GetCompilerPath());
+            if (!Directory.Exists(haxePath) && !File.Exists(haxePath))
+            {
+                ErrorManager.ShowInfo(TextHelper.GetString("Info.InvalidHaXePath"));
+                return;
+            }
```

Comment:
Yes, adding support for other package managers would be a separate task of course. Just commenting of an improvement it can bring to just extract the haxelib resolution for both parts of code (which that one could be done now).

At any rate I have the feeling decoupling from Haxelib may be really needed in the future. Also, in my company, and I'm sure in others, we have our own private Haxelib repository, so for example this PR may be a bit of a pain in the ass right now with the `MessageBox` (of course it's a nice addition and for most people welcome).